modules/dns_nsupdate/dns_nsupdate_main.rb

```diff
@@ -69,6 +80,10 @@ def remove_ptr_record(ip)
       do_remove(ip, "PTR")
     end
 
+    def remove_cname_record(fqdn_alias)
```

Comment:
https://en.wikipedia.org/wiki/CNAME_record also refers to it as name (and value). https://tools.ietf.org/html/rfc2181 refers to alias name. Since the rest of the code base uses fqdn let's leave it at fqdn_alias for now. At some point I'm going to write some patches to remove some duplication in this code anyway, let's fix it there.